mathics_scanner/generate/build_tables.py

```diff
@@ -97,6 +97,13 @@ def compile_tables(data: dict) -> dict:
         if "esc-alias" in v
     }
 
+    # AMSTEX operators
+    amstex_characters = {
+        v["wl-unicode"]: v.get("amslatex", v.get("wl-unicode"))
+        for v in data.values()
+        if "amslatex" in v and "wl-unicode" in v
```

Comment:
Shouldn't "and" be "or"? 

That is, we have an amslatex character if we have _either_ amslatex noted in an character entry _or_ a `wl-unicode character`. 

Also, it looks to me like we need to separate input from output. In input it is okay to use wl-unicode, but on _output_ we should use unicode since we don't have Wolfram front-ends.